src/models/ensemble.jl

```diff
@@ -260,7 +266,7 @@ const ExtraTreesRegressor_ = sken(:ExtraTreesRegressor)
     min_samples_leaf::Union{Int,Float64}   = 1::(_ > 0)
     min_weight_fraction_leaf::Float64      = 0.0::(_ ≥ 0)
     ## TODO: Remove the "auto" option in python sklearn v1.3
-    max_features::Union{Int,Float64,String,Nothing} = "sqrt"::(_ === nothing || (isa(_, String) && (_ in ("auto","sqrt","log2"))) || _ > 0)
+    max_features::Union{Int,Float64,String,Nothing} = 1.0::(_ === nothing || (isa(_, String) && (_ in ("auto","sqrt","log2"))) || (_ isa Number && _ > 0))
     max_leaf_nodes::Option{Int}    = nothing::(_ === nothing || _ > 0)
```

Comment:
This is a breaking change and we already tagged a breaking release. I guess we consider this a "bug fix"?

Reply:
We haven't released a breaking release yet